fleet-tenancy/config/backend-bookstore-reposync.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: configmanagement.gke.io/v1
+kind: NamespaceSelector
+metadata:
+  name: backend-bookstore
+spec:
+  mode: dynamic
+  selector:
+    matchLabels:
+      fleet.gke.io/fleet-scope: backend       # Change the value of this label selector to match different scopes by scope name
+      kubernetes.io/metadata.name: bookstore  # Change the value of this label selector to match different namespaces by namespace name
+---
+kind: RepoSync
+apiVersion: configsync.gke.io/v1beta1
+metadata:
+  name: repo-sync
+  annotations:
+    configmanagement.gke.io/namespace-selector: backend-bookstore  # Actuate this resource in the namespaces with labels matched by the namespace selector
+spec:
+  sourceFormat: unstructured
+  git:
+    repo: https://github.com/GoogleCloudPlatform/anthos-config-management-samples
+    branch: main
+    dir: fleet-tenancy/teams/backend/bookstore
+    auth: none
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: be-bookstore
+  annotations:
+    configmanagement.gke.io/namespace-selector: backend-bookstore  # Actuate this resource in the namespaces with labels matched by the namespace selector
+subjects:
+- kind: ServiceAccount
+  name: ns-reconciler-bookstore
+  namespace: config-management-system
+roleRef:
+  kind: ClusterRole
+  name: admin
```

Comment:
This doesn't seem to be the best practice. Should we create a Role and update the roleRef to reference it?

Reply:
Discussed offline. The RoleBinding will restrict the RBAC permissions to be namespace-scoped.